.github/workflows/checks.yml

```diff
@@ -390,6 +411,13 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [ "3.8", "3.9", "3.10", "3.11" ]
+    env:
+      INTEGRATION_NODE_URL: ${{ secrets.INTEGRATION_NODE_URL }}
+      TESTNET_NODE_URL: ${{ secrets.TESTNET_NODE_URL }}
```

Comment:
Maybe we should use the same naming in starknet.py and starknet-jvm, to avoid confusion in case someone uses both?
It's `INTEGRATION_RPC_URL` in jvm, we could decide on either one and use it in both SDKs.